### PR TITLE
Update architect orb

### PR DESCRIPTION
**Pull request by MarcelMue**
## Checklist

- [ ] Update changelog in CHANGELOG.md.

### CODE REVIEW

**Comment by MarcelMue** [comment]
I am not sure about many of these changes. Maybe pawel has some more insights into the use of go routines here.

File: histogramvec/histogramvec_test.go [hunk 1]
```diff
@@ -344,11 +346,11 @@ func Test_Concurrency(t *testing.T) {
 	var wg sync.WaitGroup
 	for i := 0; i < 1000; i++ {
 		wg.Add(1)
-		go func() {
```
Comment (stone-z) [inline histogramvec/histogramvec_test.go:347]: Just curious if you remember what the error was here? It seems like this makes it not concurrent anymore

Comment (MarcelMue) [reply histogramvec/histogramvec_test.go:347]: The issue is that `t.Fatal` should not be called in the routine.

Comment (stone-z) [reply histogramvec/histogramvec_test.go:347]: I see, but this then is no longer concurrent. Maybe [passing the error out on a channel](https://github.com/ipfs/go-ipfs/issues/2043#issuecomment-164136026) would be an option?

**Comment by TheoBrigitte** [comment]
Here is another attempt at upgrading the orb and some dependencies and also fixing the ci/linter issues https://github.com/giantswarm/exporterkit/pull/37/files

**Pull request closed without merging**